### DISCUSSION TITLE
test de la box Sonoff Ihost - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#29] [Comments]
# test de la box Sonoff Ihost - Haade.fr

La box Sonoff Ihost vaut le détour, avec une compatibilité matérielle impressionnante, une qualité de fabrication exceptionnelle, Sonoff signe une fois de plus un produit incontournable.

https://haade.fr/fr/blog/test-et-deballage-passerelle-sonoff-ihost-RV1126

**Comment by tommcol** [comment]
Bonjour, article très intéressant. Vous parlé d'un hack pour installer Home Assistant. Mais je n'ai rien trouvé dans l'article. Avez vous refait des test il y à t'il encore de évolutions ?
Merci

**Reply by VE2JN** [reply]
je cherche aussi ?

**Comment by haade-administrator** [comment]
salut tu trouveras tout ce qu'il faut savoir sur le github [Darkxst](https://github.com/darkxst/ha-operating-system/wiki/HAOS-on-Sonoff-iHost)
